src/herbivoresAndCarnivores.js

```diff
@@ -1,15 +1,48 @@
 'use strict';
 
 class Animal {
-  // write your code here
+  static alive = [];
+
+  constructor(name) {
+    this.name = name;
+    this.health = 100;
+    Animal.alive.push(this);
+  }
+
+  checkHealth() {
+    if (this.health <= 0) {
+      this.removeFromAlive();
+    }
+  }
+
+  removeFromAlive() {
+    Animal.alive = Animal.alive.filter((animal) => animal.health > 0);
+  }
+
+  bite(damage) {
+    this.health -= damage;
+    this.checkHealth();
+  }
```

Comment:
The `bite` method should not be implemented in the `Animal` class, as the task specifies that only `Carnivore` class should have a `bite` method. Remove this method from the `Animal` class to adhere to the task requirements.